{{cookiecutter.project_slug}}/requirements/production.txt

```diff
@@ -12,6 +12,9 @@ Collectfast==2.1.0  # https://github.com/antonagestam/collectfast
 {%- if cookiecutter.use_sentry == "y" %}
 sentry-sdk==0.14.3  # https://github.com/getsentry/sentry-python
 {%- endif %}
+{%- if cookiecutter.use_docker == "n" and cookiecutter.windows == "y" %}
+hiredis==1.0.1  # https://github.com/redis/hiredis-py
```

Comment:
I am a bit confused here. The line tells me that if people are _not_ using docker **and** they have said yes to windows, we force the use of `hiredis` in production? Is this intentional?

Reply:
Yes, it's just wrong to use a Windows machine to deploy a server. If you take a look at base.txt, it has a different condition for if you use Windows or Docker. If you do use Windows but not docker, you cannot build hiredis. But, since you wanted to use Redis, you'll also get hiredis for production, assuming, with hope, that you deploy with a unix.

However, if you use Docker on Windows, you should be able to build hiredis, and thus it can go into base.txt.

Reply:
Ah, thanks - that clarifies a lot. I see this is mentioned in the description as well but it didn't click until now.

Thanks a lot!